src/joint.dia.cell.js

```diff
@@ -608,6 +608,14 @@ joint.dia.Cell = Backbone.Model.extend({
     isLink: function() {
 
         return false;
+    },
+
+    startBatch: function(name, opt) {
+        return this.trigger('batch:start', _.extend({}, opt, { element: this, batchName: name }));
```

Comment:
Batches makes sense only if an element is part of a graph. I would take advantage of the fact, that `startBatch` is already defined on the graph. i.e.
```javascript
if (this.graph) {
  return this.graph.startBatch(name, _.extend({ element: this }, opt));
}
```